LFS171x/docs/introduction-to-hyperledger.md

```diff
@@ -1 +1,386 @@
+[Some rights reserved](https://creativecommons.org/licenses/by-nc-nd/4.0/)
+
+## Interoperability between Hyperledger Frameworks
+[![Interoperability between Hyperledger Frameworks](../images/video-image.png)](https://youtu.be/psGcqCfplMI)
+[Some rights reserved](https://creativecommons.org/licenses/by-nc-nd/4.0/)
+
+# Knowledge Check
+
+## Knowledge Check 2.1
+<!-- D -->
+Which consensus algorithms do Hyperledger frameworks support?
+
+<ol type="A">
+  <li>Kafka, RBFT, PoW and YAC</li>
+  <li>Proof of Work, Proof of Stake, Proof of Elapsed Time </li>
+  <li>YAC, Kafka, RBFT, Proof of Stake</li>
+  <li>RBFT, YAC, Proof of Elapsed Time and Kafka</li>
+</ol>
+
+## Knowledge Check 2.2
+<!-- B -->
+Which of the following are Hyperledger frameworks?
+
+<ol type="A">
+  <li>Cello, Ethereum, Indy, Burrow, Fabric</li>
+  <li>Fabric, Sawtooth, Indy, Burrow, Iroha</li>
+  <li>Sawtooth, Indy, Fabric, Cello, Composer</li>
+  <li>Burrow, Iroha, Fabric, Sawtooth, Cello</li>
+</ol>
+
+## Knowledge Check 2.3
+<!-- C -->
+Which of the following best defines Hyperledger Sawtooth?
+
+<ol type="A">
+  <li>A production distributed ledger</li>
+  <li>A blockchain framework for building, deploying, and running Docker containers</li>
+  <li>A modular blockchain framework that implements the PoET consensus algorithm</li>
+  <li>A distributed ledger platform designed for supply chain solutions </li>
+</ol>
+
+## Knowledge Check 2.4
+<!-- D -->
+What is Indy?
+
+<ol type="A">
+  <li>A business blockchain framework for supporting independent identity on distributed ledgers</li>
+  <li>A Hyperledger project that allows digital identities to be interoperable across administrative domains, applications, and any other silo</li>
+  <li>A Hyperledger project that provides tools, libraries, and reusable components for providing digital identities rooted on blockchains or other distributed ledgers </li>
+  <li>All of the above</li>
+</ol>
+
+# Conclusions & Learning Objectives (Review)
+
+## Learning Objectives (Review)
+You should now be able to:
+
+* Explain the differences between Hyperledger and permisionless blockchain technologies.
+* Discuss how Hyperledger leverages open standards and open governance to support business solutions.
+* Discuss Hyperledger frameworks (Iroha, Sawtooth, Fabric, Indy, and Burrow) and modules (Cello, Explorer, and Composer).
+
+## Blockchain Security at Hyperledger (David Huseby)
+[![Blockchain Security at Hyperledger (David Huseby)](../images/video-image.png)](https://youtu.be/BOLNop07nTU)
+[Some rights reserved](https://creativecommons.org/licenses/by-nc-nd/4.0/)
+
+## How Will Hyperledger Change the Blockchain Ecosystem? (Brian Behlendorf)
+[![How Will Hyperledger Change the Blockchain Ecosystem? (Brian Behlendorf)](../images/video-image.png)](https://youtu.be/_pxGcCnYVlk)
+[Some rights reserved](https://creativecommons.org/licenses/by-nc-nd/4.0/)
```

Comment:
Remove - the course is actually licensed under https://creativecommons.org/licenses/by/4.0/. Unfortunately, EdX doesn't let you specify which Creative Common license for videos. Therefore, we will let the overall license cover these.